### PR TITLE
remove subtasks from board

### PR DESCRIPTION
**Pull request by Lamperoyge**
https://www.loom.com/share/53615b13fbf84997a53638e21f2f5f73

https://github.com/wondrous-dev/wondrous-backend/pull/404

### CODE REVIEW

**Comment by terryli0095** [comment]
are we just hiding subtasks? should we instead use the backend query? i actually don't have strong opinion. only concern is pagination

**Comment by Lamperoyge** [comment]
> are we just hiding subtasks? should we instead use the backend query? i actually don't have strong opinion. only concern is pagination

We are using the BE flag ( https://github.com/wondrous-dev/wondrous-backend/pull/404 ) this PR is an improvement because on FE every time you added a subtask  FE was updating the board too and was adding the subtask to local state, this way you’d see the subtask when creating one. Also a different improvement is to render subtasks content in the view only when there is a taskId, there were a couple of requests that were failing because of missing props.

tldr: we do use BE flag, changes here are improvements when creating task and viewing subtasks, for fetching we fully rely on BE include_sub_task flag
